### PR TITLE
gene/tales-of-privacy-1

### PR DESCRIPTION
**Pull request by ajingene**
raw text version

### CODE REVIEW

**Review by josipkoncurat — CHANGES_REQUESTED** [review]
The post is nicely put, It gave some nice insight on privacy in Bitcoin and how it was achieved in Monero. Do you think governments will ever allow private blockchains to be used freely? 
The current system is far away from real privacy, this would be a big shift.

Check out my comments but anyway well done!

File: _posts/2021-04-23-gene-tales-of-privacy-1.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+you, a person with an actual name and street address.  Meaning you can
+make a financial transaction without your personal details being
+written down anywhere in it.  However, that doesn't make bitcoin
+actually private in many a use case.  Because, as stated at the start
+of this post, bitcoin is a *public* blockchain.  All records of
+transactions are visible to anyone who knows where to look.  This
+means a couple of things:
+1. If you make your personal data known to the other party involved in
+   a transaction with you, all your transactions can be traced back to
+   you if that other party reveals your data.
```
[the remark this answers is not in the document]
Comment (ajingene) [reply _posts/2021-04-23-gene-tales-of-privacy-1.html:21]: KYC in general. For example if you're buying products over the Internet, (like on a webshop that supports bitcoin). Maybe in some cases it's enough to just leave an email address, but in most cases you'd have to write your name and street address.

File: _posts/2021-04-23-gene-tales-of-privacy-1.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+without any outsider knowing what you bought, from who at what price?
+You can with cash.  How about bitcoin?
+
+In a bitcoin transaction, bitcoin is moved from one bitcoin address to
+another.  Let's say you're a recepient in such a transaction.  Your
+bitcoin address is just a random string of characters unrelated to
+you, a person with an actual name and street address.  Meaning you can
+make a financial transaction without your personal details being
+written down anywhere in it.  However, that doesn't make bitcoin
+actually private in many a use case.  Because, as stated at the start
```
[the remark this answers is not in the document]
Comment (ajingene) [reply _posts/2021-04-23-gene-tales-of-privacy-1.html:15]: It means the same thing as what you wrote, it's just in a different style. I would say "many a use case" is more elegant and poetic. I put it there "teasingly", to loosen the atmosphere. Becasuse an elegant phrase is unexpected for this subject.

File: _posts/2021-04-23-gene-tales-of-privacy-1.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+cash?  In particular, can you go and shop to your heart's content
+without any outsider knowing what you bought, from who at what price?
+You can with cash.  How about bitcoin?
+
+In a bitcoin transaction, bitcoin is moved from one bitcoin address to
+another.  Let's say you're a recepient in such a transaction.  Your
+bitcoin address is just a random string of characters unrelated to
+you, a person with an actual name and street address.  Meaning you can
+make a financial transaction without your personal details being
+written down anywhere in it.  However, that doesn't make bitcoin
```
[the remark this answers is not in the document]
Comment (ajingene) [reply _posts/2021-04-23-gene-tales-of-privacy-1.html:14]: ok

File: _posts/2021-04-23-gene-tales-of-privacy-1.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+   those addresses is used, as not everyone keeps this hidden in the
+   first place.  There are companies that specialize in this kind of
+   analytics and there's plenty money and interest in this -- in
+   other words this is a reality, not a possibility.
+
+For all you folks that have not much interest in anonymity, but are
+still reading for some reason, I'll give you a more tangible reason to
+continue: lack of anonymity in bitcoin makes it /not really fungible/.
+You see, 1BTC that's freshly minted != 1BTC that has a criminal
+transaction history (criminal in the literal sense).  The freshly
```
[the remark this answers is not in the document]
Comment (ajingene) [reply _posts/2021-04-23-gene-tales-of-privacy-1.html:37]: Yeah, I'll edit it in the non-raw-text version to look better.

File: _posts/2021-04-23-gene-tales-of-privacy-1.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+minted coin is worth more to a potential buyer because it comes with
+no history (no *bad* history).  For example, if coins were stolen from
+its owner at some point, it's not impossible that authorities will
+attempt to return those coins to the original owner.  The risk of this
+makes them less attractive to the prospective bitcoin buyer.
+
+So is there a way to use bitcoin anonymously?  Why it's simple of
+course!  You buy it in cash, don't leak your details ever, and trade
```
[the remark this answers is not in the document]
Comment (ajingene) [reply _posts/2021-04-23-gene-tales-of-privacy-1.html:45]: It's the same as saying "It's simple, of course!"
In general, "why of course" == "of course".  Again, it sounds a bit more elegant. Like a fat middle aged British man is saying it with a smile.

**Comment by ajingene** [comment]
>Do you think governments will ever allow private blockchains to be used freely?
As long as they can hack the system (break it, get around the privacy), most governments probably won't ban it. But the question is also how to ban it effectively and can it really be done.

**Comment by ajingene** [comment]
did some changes

**Pull request merged**